### PR TITLE
Fix Save to not open dialog if you opened a project file

### LINKED ISSUE #711: Save button is actually Save As
Even when File/Save does an immediate save, the Save button always pops up the Save dialog box.

### PR DESCRIPTION
**Pull request by jaredly**
This depends on https://github.com/meerk40t/meerk40t/pull/763, to make a distinction between "open" and "import" (because when we import a file, we don't want to save back to it).
fixes #711

### CODE REVIEW

**Review by tatarize — APPROVED** [review]

File: meerk40t/gui/wxmribbon.py [hunk 1]
```diff
@@ -214,7 +214,7 @@ def open_simulator(v=None):
         )
         toolbar.Bind(
             RB.EVT_RIBBONBUTTONBAR_CLICKED,
-            lambda e: self.context(".dialog_save\n"),
+            lambda e: self.context(".on_click_save\n"),
```
Comment (Sophist-UK) [inline meerk40t/gui/wxmribbon.py:217]: I don't think this will work as there isn't a console command registered for `on_click_save`.

**Review by Sophist-UK — CHANGES_REQUESTED** [review]
Since this includes the changes for #763 I would suggest that you close #763 and update the title and description of this one.

**Comment by Sophist-UK** [comment]
P.S. Apologies for having missed you off the new contributors list on Facebook. I have rectified this.

File: meerk40t/gui/wxmribbon.py (hunk 1 above)
Comment (jaredly) [reply meerk40t/gui/wxmribbon.py:217]: hmm it's definitely working for me, I thought it just keyed off of the method name

Comment (Sophist-UK) [reply meerk40t/gui/wxmribbon.py]: Yup.

**Review by Sophist-UK — APPROVED** [review]
LGTM

**Review by tatarize — APPROVED** [review]

**Pull request merged**